### PR TITLE
72 i can edit an individual skill hq(front-end)

### PR DESCRIPTION
**Pull request by fadeomar**
#### Issues this PR relates to
#72 
---

#### Any info the reviewer needs to know to test 
e.g. Add env vars, run npm install

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

---

#### Quick Pull Request Checklist
Please make sure you ensure the following has been done before marking your PR as awaiting-review. For full checklist and process review, please refer to Process Guide (#8)
- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- The new work has been checked with Axe accessibility checker and doesn't throw significant errors  
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs

### CODE REVIEW

File: client/src/components/SkillModal/style.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  .ant-drawer-content-wrapper {
+    box-shadow: none !important;
+  }
+  .ant-drawer-content {
+    background: transparent;
+  }
+  .ant-drawer-body {
+    display: flex;
+    align-items: center;
+    overflow-y: hidden;
```
Comment (RamyAlshurafa) [inline client/src/components/SkillModal/style.js:14]: make it `auto` so its scrollable on mobiles

File: client/src/components/SkillModal/style.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    ${({ theme }) => theme.media.mobile} {
+      padding: 0;
+    }
+  }
```
Comment (RamyAlshurafa) [inline client/src/components/SkillModal/style.js:18]:
```suggestion
```

**Review by RamyAlshurafa — CHANGES_REQUESTED** [review]

File: client/src/components/Inputs/Textarea/style.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  :focus {
+    border-color: ${({ theme }) => theme.colors.white};
+    border-bottom: 3px solid;
+  }
+  `}
+`;
+
+export const Textarea = styled.textarea`
+  ${setMargin}
+  ${CS.placeholderStyle}
```
[the remark this answers is not in the document]
Comment (fadeomar) [reply client/src/components/Inputs/Textarea/style.js:41]: done for all  `${setMargin}` without the semicolon

File: client/src/components/SkillModal/style.js (hunk 1 above)
Comment (fadeomar) [reply client/src/components/SkillModal/style.js:14]: done

File: client/src/components/SkillModal/style.js (hunk 2 above)
Comment (fadeomar) [reply client/src/components/SkillModal/style.js:18]: done

File: client/src/components/SkillModal/index.js [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+            name="task3"
+            value={state.task3}
+            handleChange={(e) => handleChange('task3', e.target.value)}
+            disabled={!state.task2 || state.hideSkill}
+            error={errors.task3}
+          />
+
+          {/* submit */}
+          <Button
+            style={{ marginTop: 54, border: 'none' }}
```
[the remark this answers is not in the document]
Comment (fadeomar) [reply client/src/components/SkillModal/index.js:151]: done

File: client/src/components/Inputs/Textarea/index.js [hunk 5]
```diff
@@ -0,0 +1,81 @@
+import * as T from '../../Typography';
+import * as CS from '../style';
+import * as S from './style';
+
+const BasicInput = ({
```
[the remark this answers is not in the document]
Comment (fadeomar) [reply client/src/components/Inputs/Textarea/index.js:5]: done

File: client/src/components/SkillModal/index.js [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+          {/* Description */}
+          <Textarea
+            type="textarea"
+            name="description"
+            rows="3"
+            cols="33"
+            value={state.description}
+            handleChange={(e) => handleChange('description', e.target.value)}
+            label="How can this skill help in the real world?"
+            placeholder="Lots of text to go here that describes the skill and provides context on what you need to do to complete the task"
```
[the remark this answers is not in the document]
Comment (fadeomar) [reply client/src/components/SkillModal/index.js:116]: done

**Review by RamyAlshurafa — APPROVED** [review]

**Pull request merged**